sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -467,15 +483,34 @@ export interface SentimentConfidenceScores {
 // @public
 export type StringIndexType = "TextElements_v8" | "UnicodeCodePoint" | "Utf16CodeUnit";
 
+// @public
+export interface TargetConfidenceScoreLabel {
+    // (undocumented)
+    negative: number;
+    // (undocumented)
+    positive: number;
+}
+
+// @public
+export interface TargetSentiment {
+    confidenceScores: TargetConfidenceScoreLabel;
+    length: number;
+    offset: number;
+    sentiment: TokenSentimentValue;
+    text: string;
+}
+
 // @public
 export interface TextAnalyticsActionErrorResult {
     readonly error: TextAnalyticsError;
+    readonly failedOn: Date;
```

Comment:
this is the equivalent of `TextAnalyticsActionSuccessState.completedOn` but when there is an error, right?
Is it common in JS to use `failedOn` instead of `completedOn`?

Reply:
is there an example of an endpoint that returns the failure date and time of a failed action? I just thought using the word completed here is misleading. I am not even sure why the service returns this to begin with.

Reply:
They all do. See
```
{
    "jobId": "a0647292-7daf-449f-8cfa-081f3670db84",
    "lastUpdateDateTime": "2021-03-08T14:54:37Z",
    "createdDateTime": "2021-03-08T14:54:36Z",
    "expirationDateTime": "2021-03-09T14:54:36Z",
    "status": "failed",
    "errors": [
        {
            "code": "InvalidRequest",
            "message": "Job task parameter value phi is not supported for model-version parameter for job task type KeyPhraseExtraction. Supported values latest,2020-07-01.",
            "target": "#/tasks/keyPhraseExtractionTasks/0"
        }
    ],
    "tasks": {
        "details": {
            "lastUpdateDateTime": "2021-03-08T14:54:37Z"
        },
        "completed": 0,
        "failed": 1,
        "inProgress": 0,
        "total": 1,
        "keyPhraseExtractionTasks": [
            {
                "lastUpdateDateTime": "2021-03-08T14:54:37.8747764Z",
                "state": "failed"
            }
        ]
    }
}
```

And to clarify, your property is mapping to this area, right?
```
"keyPhraseExtractionTasks": [
            {
                "lastUpdateDateTime": "2021-03-08T14:54:37.8747764Z",
                "state": "failed"
            }
        ]
```

Reply:
@maririos I think you misunderstood my comment. I meant to ask if you have any example of endpoints outside TA that does something similar so I can go and look at what the JS SDK does for naming such date property.

> And to clarify, your property is mapping to this area, right?

Yes!

Reply:
Ahh I see, I thought in TA.
I currently can't think of any but maybe looking on the API view of other libraries in JS might give u a hint